src/views/shared/transaction-info/transaction-info.view.jsx

```diff
@@ -42,6 +43,10 @@ function TransactionInfo({
     if (txData.state === TxState.Forged) {
       return { subtitle: TxStatus.Confirmed };
     }
+    console.log(txData);
+    if (txData.errorCode) {
```

Comment:
This should probably go in a different PR

Reply:
yes